### ISSUE TITLE
Code stype

### ISSUE THREAD

**Issue by kleinicke** [yijingru/BBAVectors-Oriented-Object-Detection#99]
I really like your work. But you could further improve the code by following a few guidelines.
1. Naming: 
A lot of variables are very hard to guess. You have the space to use longer variable names, so use it.
Especially when they were described in detail within the paper.
wh -> boxparams (exept of one occurence where it stands correctly for wight hight), reg -> offset, ...

2. The flow of data is complicated:
The data gets often stored in variables, unpacked, transformed ...
This makes it very hard to follow. You could probably have less functions that do more in one place without the need to completely transform reshape the data for the next function. 
Especially confusing is the use of self.decoder.ctdet_decode, func_utils.decode_prediction and nms in test.py 
These three things could be combined into one function.

**Comment by yijingru** [comment]
Thanks! I will consider improving the code.
